src/lib/features/events/event-service.ts

```diff
@@ -110,4 +142,48 @@ export default class EventService {
             });
         }
     }
+
+    convertToQueryParams = (
```

Comment:
Not a big deal, but the name here is confusing, because we're accepting query params but we're also returning query ... params?

Is one of them URL query params, and the other is db query params?

Maybe we could do something like `toDbQuery` or something?